### ISSUE TITLE
Overall status! 

### ISSUE THREAD

**Issue by Zeyadas** [koreader/koreader#7082]
I was wondering if you could add an (overall reading status), I'll try to describe it as I see it. 

The overall reading status would be similar to the (book status) but it will be for:

1- the book titles you've read. 
2- book ratings. 
3- reading time for each book. 
4- read page for each book. 
5- book status (finished - reading - on hold) 
5- total reading time. 

You can choose what to view on the overall status (example: only the book title and reading time for each book and total reading time) 

You can view it as (daily - weekly - monthly - yearly), or maybe you can choose specific dates. 

You can add a (chart) depending on the information collected above.

You can somehow share it (excel document or pdf or something). 
![Screenshot_2020-12-29-16-20-24](https://user-images.githubusercontent.com/70109540/103331913-0fb5c100-4a79-11eb-9a40-a0488810f78e.png)

**Comment by NiLuJe** [comment]
That'd be a bit unwieldy right now, because while some of the info is the stats database, some of it is also in each individual sidecar metadata.

What can "easily" be done is essentially something similar to the Calendar view (e.g., purely time-based data).

**Comment by Zeyadas** [comment]
That would be great as a start.. 
And I was thinking about using koreader in (landscape mood).

**Comment by NiLuJe** [comment]
Mostly everything supports Landscape now, so I'm not sure if there was a question in there? ;).

**Comment by Zeyadas** [comment]
Using koreader in landscape mode didn't work for me. Didn't rotate.
I'm using an android tablet.

**Comment by NiLuJe** [comment]
Then we're going to need way more details about said tablet ;).

**Comment by poire-z** [comment]
Just asking, because it's not obvious from your words that you know it exists: do you know we have a Reading Statistics plugin ? That you can enable/show via the top Tools menu.
Except for book ratings and status, it has everything you asked for.

**Comment by Zeyadas** [comment]
It's really there the (reading statistics), you guys are great at what you do.
I didn't know that It was there all the time. 
![Screenshot_2020-12-30-21-35-58](https://user-images.githubusercontent.com/70109540/103373935-4de9c980-4ae7-11eb-8d43-e91ad2a6805d.png)

I wish if I can add a (tag) or something for the finished books.
And for more details about m tablet it's Samsung galaxy Tab A 6 (2016) (SM-T285)

**Comment by pazos** [comment]
@Zeyadas: autorotation is disabled. To rotate you'll need to do it manually. (Ie: on the reader, from the bottom menu, on the FM -> from top menu -> settings -> screen)

**Comment by Zeyadas** [comment]
The rotation.. Found it. 
Again, you are great guys, now I'm almost convinced that this app is perfect.
Thank you all.

**Comment by Zeyadas** [comment]
Is there a full (user guide - manual) as a pdf or something that describes everything in the app? I promise I'll read it even though it's not an Arabic tradition (reading manuals).

**Comment by Zeyadas** [comment]
And I noticed something, in landscape mode the dual page (on), the dual page in Arabic text should be inverted. (flipped) 
![Screenshot_2020-12-30-22-14-41](https://user-images.githubusercontent.com/70109540/103376080-bc7d5600-4aec-11eb-901f-d67a0ca00e30.png)
In this screenshot the text starts on the left page and continues on the right page, that should be inverted somehow.

**Comment by poire-z** [comment]
> in landscape mode the dual page (on), the dual page in Arabic text should be inverted. (flipped)

Yep :/ That's one of the few things that were really hard to do when we did RTL support. Because it's the EPUB engine (crengine) job (I guess there's some metadata in the EPUB .opf that says something about reading order) - but also lots of stuff on the UI side with text selection crossing pages.
So, I guess for the moment, RTL readers should just not use "dual pages" :/

**Comment by Zeyadas** [comment]
I'm so grateful for your responses guys.

**Comment by mergen3107** [comment]
> Is there a full (user guide - manual) as a pdf or something that describes everything in the app? I promise I'll read it even though it's not an Arabic tradition (reading manuals).

There is KOReader Wiki here on GitHub. It is filled with volunteer users, so it might not be complete
